exercises/strain/strain_test.go

```diff
@@ -19,6 +19,14 @@ import (
 	"testing"
 )
 
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
```

Comment:
Ah ok, no worries, I'm happy either way tbh, I'll fix this PR, as it's probably easier than going back over what's already been done, unless anyone has any specific objections?